### PR TITLE
docker: use content classification model from ML repo releases

### PR DESCRIPTION
**Pull request by cyberj0g**
**What does this pull request do? Explain your changes. (required)**
Change detection model URL in Dockerfile to point to livepeer-ml repo release

**Specific updates (required)**
- 

**How did you test each of these updates (required)**
Tested with [LPMS](https://github.com/livepeer/lpms/pull/269) scenedetection 

**Does this pull request close any open issues?**

**Checklist:**

- [x] Read the [contribution guide](./doc/contributing.md)
- [x] `make` runs successfully
- [x] All tests in `./test.sh` pass
- [x] README and other documentation updated
- [x] [Pending changelog](./CHANGELOG_PENDING.md) updated

### CODE REVIEW

File: docker/Dockerfile.build-linux [hunk 1]
```diff
@@ -30,4 +30,4 @@ ENV GOPATH /go
 RUN mkdir -p /go
 WORKDIR /build
 
-RUN curl -LO https://storage.googleapis.com/lp_testharness_assets/tasmodel.pb
\ No newline at end of file
+RUN curl -LO https://github.com/livepeer/livepeer-ml/releases/download/v0.1/tasmodel.pb
```
Comment (yondonfu) [inline docker/Dockerfile.build-linux]: Is there no functional difference between v0.1 and v0.2/v0.3 listed in https://github.com/livepeer/livepeer-ml/releases ?

**Review by yondonfu — COMMENTED** [review]
Just one comment about model version to use.

File: docker/Dockerfile.build-linux (hunk 1 above)
Comment (cyberj0g) [reply docker/Dockerfile.build-linux]: None, it was test runs to make sure the runner is fully functional. I'll change it to point to 0.3.

**Review by yondonfu — APPROVED** [review]
LGTM

**Pull request merged**